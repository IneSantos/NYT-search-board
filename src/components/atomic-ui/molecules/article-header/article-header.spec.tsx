import React from "react";
import { render } from '@testing-library/react';
import ArticleHeader from "./article-header";

describe("ArticleHeader component tests", () => {
  test(`ArticleHeader component render`, () => {
    const { container } = render(
      <ArticleHeader headline={"This is an headline"} date="21/11/2021"/>
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});
