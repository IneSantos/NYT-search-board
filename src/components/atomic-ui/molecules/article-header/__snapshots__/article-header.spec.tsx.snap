// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArticleHeader component tests ArticleHeader component render 1`] = `
<div
  class="article-header-wrapper"
>
  <a
    class="article-action"
    href="/"
  >
     
    &lt; Go to results page
     
  </a>
  <div
    class="article-header"
  >
    <h3
      class="title"
    >
      This is an headline
    </h3>
    <div
      class="subtitle"
    >
      NaN.NaN.NaN
    </div>
  </div>
</div>
`;
